modules/common/src/main/java/org/opencastproject/security/api/AccessControlList.java

```diff
@@ -68,6 +68,13 @@ public List<AccessControlEntry> getEntries() {
     return entries;
   }
 
+  public boolean isValid() {
+    for (AccessControlEntry entry : entries) {
+      if (!entry.isValid()) return false;
+    }
+    return true;
```

Comment:
Maybe a bit simpler?
```suggestion
    return entries.stream().allMatch(AccessControlEntry::isValid);
```